integration_tests/tests/consistency/consistency_daily_shop.sql

```diff
@@ -0,0 +1,77 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with prod as (
+    select
+        date_day,
+        shop_id, 
+        source_relation,
+        sum(count_orders) as count_orders,
+        count(count_customers) as count_customers,
+        sum(order_adjusted_total) as order_adjusted_total,
+        sum(count_abandoned_checkouts) as count_abandoned_checkouts,
+        sum(count_fulfillment_attempted_delivery) as count_fulfillment_attempted_delivery,
```

Comment:
This validation must have the `shopify_using_fulfillment_event` variable defined as `true` (default: `false`) in order for the validation to succeed.

Would you either be able to put a conditional in here to ensure the validation runs as expected regardless of the variable being true/false. Or leave a comment to callout that the `shopify_using_fulfillment_event` variable must be `true` in order for this to succeed.

Reply:
![image](https://github.com/fivetran/dbt_shopify/assets/74217849/3213ca90-8899-4934-8a3f-023b0e5cdb95)

Reply:
ah i'll add a conditional